### DISCUSSION TITLE
bootstrap of step-ca fails

### DISCUSSION THREAD

**Discussion by gfre** [goreleaser/goreleaser#5218] [Q&A] [answered]
When calling `make bootstrap` with go1.22 for **step-ca** [v0.27.5](https://github.com/smallstep/certificates/releases/tag/v0.27.5) I'll get the the error
`go: downloading github.com/multiformats/go-base32 v0.1.0
../go/pkg/mod/github.com/multiformats/go-multibase@v0.2.0/base32.go:4:2: github.com/multiformats/go-base32@v0.1.0: verifying module: github.com/multiformats/go-base32@v0.1.0: TileReader returned bad result slice (tile/8/1/183 len=4096, want 8192)
make: *** [Makefile:24: bootstrap] Error 1`

makefile line 24 does the call `$Q go install github.com/goreleaser/goreleaser@latest`.

Can anyone please verify if it's really a bug or not, and in case give me a hand to get it sorted. Cheers.

**Comment by caarlos0** [comment]
> Can anyone please verify if it's really a bug or not, and in case give me a hand to get it sorted.

can't reproduce this... never seen that kind of error either

**Reply by gfre** [reply, marked as answer]
Sorry, for the late reply. I did a clean re-build of my lxc and it worked afterwards.  Probably ruined something during the first attempt. Cheers.
